src/model_DAforcing.py

```diff
@@ -369,10 +372,22 @@ def run(self, values: dict):
                 self._t0,
                 lastobs_output_folder
             )
+  
+        # if output_parameters.get('stream_output',{}) is not None and output_parameters.get('stream_output',{}).get('qvd_nudge'):
+        if output_parameters.get('stream_output',{}):
+            
+            t0 = self._t0
+            stream_output_directory =  output_parameters.get('stream_output').get('stream_output_directory')
+            stream_output_type = output_parameters.get('stream_output').get('stream_output_type')
+            stream_output_internal_frequency = output_parameters.get('stream_output').get('stream_output_internal_frequency')
             
-        if output_parameters.get('stream_output',{}).get('qvd_nudge'):
-            #write flowveldepth file
-            pass
+            write_flowveldepth_netcdf(values,
+                                      stream_output_directory,   
+                                      t0,
+                                      1, 
```

Comment:
This should be defined based on how the BMI engine is being run. We think it'll likely be hourly time intervals, but it doesn't have to be.